test/es-module/test-cjs-esm-warn.js

```diff
@@ -26,7 +26,7 @@ child.on('close', common.mustCall((code, signal) => {
   assert.strictEqual(code, 1);
   assert.strictEqual(signal, null);
 
-  assert.ok(stderr.indexOf(
+  assert.ok(stderr.replace(/\r/, '').indexOf(
```

Comment:
Nit:

```suggestion
  assert.ok(stderr.replace(/\r/g, '').indexOf(
```

Reply:
Secondary nit: Instead of `indexOf(....) !== -1`, use `includes()`?

Reply:
👍

Reply:
> Secondary nit: Instead of `indexOf(....) !== -1`, use `includes()`?

Although changing to `includes()` means updating the test more than just fixing what's broken so maybe someone else can do that at a later date at a Code & Learn or whatever.

Looks good to me with or without the suggestion.